### ISSUE TITLE
Renaming/Moving a file through Windows client causes version history loss

### ISSUE THREAD

**Issue by Weak-Chicken** [nextcloud/desktop#2132]
### Expected behaviour
When rename/move a file, it shows "moved" in nextcloud activity page.

### Actual behaviour
When rename/move a file, it shows two steps, which is first "deleted" the file and then "created" it in target path in nextcloud activity page.

This cause that the original file lost all its version history information.

### Steps to reproduce
1. Create a file
2. modify it, generate some version history infomation
3. Move it to a new place with nextcloud client, and the version history will be lost

### Client configuration
Client version:

Operating system:
Windows 10 1903

OS language:
English

Qt version used by client package (Linux only, see also Settings dialog):
N/A

Client package (From Nextcloud or distro) (Linux only):
N/A

Installation path of client:
C:\Program Files\Nextcloud

### Server configuration

Nextcloud version:
19.0.0

Storage backend (external storage):
Not quite clear about this question... Plz explain this to me if it is needed

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

1. Client logfile: Output of `nextcloud --logwindow` or `nextcloud --logfile log.txt`
(On Windows using `cmd.exe`, you might need to first `cd` into the Nextcloud directory)
(See also https://docs.nextcloud.com/desktop/2.3/troubleshooting.html#log-files)

I used above command and find only this:
[OCC::Application::setupLogging 	"################## Nextcloud locale:[en_US] ui_lang:[en_GB] version:[2.6.4stable-Win64 (build 20200303)] os:[Windows 10 (10.0)]"
[OCC::Application::slotParseMessage 	Running for 23236 sec

I don't think that would be helpful. However, I think the problem I'm discussing here doesn't really need a log to explain here. Plz tell me if a log is really helpful and I'll try to figure out what was the problem with my log file.

2. Web server error log:
There is no error

3. Server logfile: nextcloud log (data/nextcloud.log):
It happens to be I have some problem on accessing nextcloud backend... Again, if this is really needed please let me know and I'll upload these logs later.

In summary, I would say that this might be a simple move/rename file problem, even not a bug maybe.

Expected behaviour:
![image](https://user-images.githubusercontent.com/19397726/85942857-ac7a9300-b95e-11ea-9e0f-e5d9ac5f5f75.png)

(This is when I move a file through nextcloud web UI)

Actual behaviour:
![image](https://user-images.githubusercontent.com/19397726/85942873-d92eaa80-b95e-11ea-9621-8fd560e53e23.png)

And this is when I do the same thing through Windows client, which causes version history loss.

(I just found renaming folder is work again....don't know why. But renaming/move files is still the same)
